### ISSUE TITLE
The MAUI Template still doesn't build because of the splash screen if you select "place project and solution in the same directory

### ISSUE THREAD

**Issue by Deus-nsf** [dotnet/maui#23562]
### Description

![image](https://github.com/dotnet/maui/assets/13644590/374bd2da-6db3-47a8-b71a-ec3ac3a78775)
![image](https://github.com/dotnet/maui/assets/13644590/6c6c3c22-f41a-4cb7-8efc-22e1d110ba99)

Just trying to launch the Windows Machine target (after rebuild).
I already created an issue a year ago regarding this, assumed it was fixed but it looks like it's here again.
Not a big deal but this is going to confuse a lot of people.

### Version with bug

9.0.0-preview.6.24327.7

### Is this a regression from previous behavior?

No, this is something new

### Last version that worked well

Unknown/Other

### Affected platforms

Windows

### Did you find any workaround?

One need to not create a MAUI app with the solution and project in the same directory, but some may prefer doing that, for example with some source control hierarchy.

**Comment by Deus-nsf** [comment]
Nope Mr bot, no identical issue in your list hehe, but I found the original issue: https://github.com/dotnet/maui/issues/14422

**Comment by Deus-nsf** [comment]
Looks like having a different folder still causes problems now? I was able to reproduce it with the most casual way.

**Comment by Deus-nsf** [comment]
Okay I figured it out, for Windows at least, I had to disable all other platforms (even though I'm not launching on anything else but with a Windows Machine target), including editing the project file because the properties UI still doesn't offer the option to disable the targetting of Mac Catalyst (so commented the <TargetFrameworks>net8.0-maccatalyst</TargetFrameworks> line).

This is really not beginner friendly, and I was able to relatively quickly get out of this because I had prior experience with issues like this.

**Comment by Deus-nsf** [comment]
It also looks like the project file completely freaks out when changing the Windows target framework and becomes corrupt, with no possibility from the UI to select a target framework again, I remember having that bug a while ago too.
I swear my Visual Studio is up to date lol.

**Comment by Deus-nsf** [comment]
I deleted everything and tried again, but this time instead of launching VS without code and creating a new project, I created a new project from the window you get before VS even launches, and then I didn't rebuild the project right off the bat and just launched a debug with the Windows Machine target, and now it works...
I then tried to rebuild and launch again and it still works. So I'm really not sure what's going on there but oh well...

**Comment by Deus-nsf** [comment]
Well nevermind, enabling / disabling platforms is still as bugged as ever, even after restarts, I will try to switch to the .NET MAUI Multi-Project App template because the basic one is just a complete mess.

**Comment by Zhanglirong-Winnie** [comment]
This issue has been verified using Visual Studio 17.11.0 Preview 3.0(.NET9.0). Can repro on windows platform. This is a known issue, unstable to repro. Solution: Clean up the project, rebuild it, run it again, and the problem disappears.

**Comment by CleverSoftwarePoland** [comment]
Try to check your Windows. "Package.appxmanifest" file and remove splash screen reference to png file, remember to clean project after you update manifest file.

**Comment by mattleibow** [comment]
Yeah... we probably can remove the splash screen from the manifest on windows because windows does not support splash screens anymore.

The issue of not generating is still there, you just won't see it fail.

But, to fix this, maybe you can install the extension and attach a binlog when it happens: https://github.com/dotnet/maui/wiki/Capturing-Binary-Logs

You can start the log collection, and as soon as the IDE has an error, send the last few builds.

**Comment by jfversluis** [comment]
I think this has been addressed in the meantime. Let us know if that is not the case!
